de/cheatsheet-deep-learning.md

```diff
@@ -1,162 +1,162 @@
 **1. Deep Learning cheatsheet**
 
-&#10230;
+&#10230; Deep Learning Spickzettel
 
 <br>
 
 **2. Neural Networks**
 
-&#10230;
+&#10230; Neuronale Netze
 
 <br>
 
 **3. Neural networks are a class of models that are built with layers. Commonly used types of neural networks include convolutional and recurrent neural networks.**
 
-&#10230;
+&#10230; Neuronale Netze sind eine Klasse von Modellen die in Schichten aufgebaut sind. Gängige Typen neuronaler Netze sind unter anderem faltende und rekurrente neuronale Netze.
 
 <br>
 
 **4. Architecture ― The vocabulary around neural networks architectures is described in the figure below:**
 
-&#10230;
+&#10230; Architektur - Das Vokabular rund um Architekturen neuronaler Netze ist in folgender Abbildung beschrieben: 
 
 <br>
 
 **5. [Input layer, hidden layer, output layer]**
 
-&#10230;
+&#10230; [Eingangsschicht, versteckte Schicht, Ausgangsschicht]
```

Comment:
According to german literature (e.g. http://wwwdh.cs.fau.de//lectures/ks2/MULTI-LAYER.pdf) it is:
[_Eingabeschicht, Verborgene Schicht, Ausgabeschicht_]
rather than 
[Eingangsschicht, versteckte Schicht, Ausgangsschicht]

Reply:
Also "Eingabeschicht and "Ausgabeschicht" instead of "Eingangsschicht" and "Ausgangsschicht" to mirror 

> output = Ausgabe

as in your other [comment](https://github.com/shervinea/cheatsheet-translation/pull/106#discussion_r272991150)?